### PR TITLE
update to README-0.1

### PR DESCRIPTION
**Pull request by javadamiri**
prepare the README file for release-0.1

### CODE REVIEW

**Comment by javadamiri** [comment]
> Only minor edits, aside from the use of submodules. I thought this was fixed long ago!
> 
> If it is at all possible to fix this, please normalize the jikesrvm repo, getting rid of the use of submodules.

I finished removing submodules and updating CI scripts and README accordingly

File: mmtk/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ lto = true
 libc = "0.2"
 lazy_static = "1.1"
 log = {version = "0.4", features = ["max_level_trace", "release_max_level_off"] }
-mmtk = { path = "../repos/mmtk-core" }
+mmtk = { git = "ssh://git@github.com/mmtk/mmtk-core.git" }
```
Comment (qinsoon) [inline mmtk/Cargo.toml:18]: You would need a `rev` to specify the revision. You can check the `Cargo.toml` for OpenJDK: https://github.com/mmtk/mmtk-openjdk/blob/7dd5ef78f537b197700feec913890e7d415b16dc/mmtk/Cargo.toml#L24

With this change, the CI config needs to be updated as well, as the CI runner cannot pull the repo by `ssh`. Check the step here on OpenJDK CI (https://github.com/mmtk/mmtk-openjdk/blob/7dd5ef78f537b197700feec913890e7d415b16dc/.github/workflows/ci.yml#L19).

Also in mmtk-core, there are a few places you would need some change. Check the difference between JikesRVM and OpenJDK in mmtk-core CI, and make the JikesRVM steps similar to the OpenJDK's. 

Make sure that CI on both mmtk-jikesrvm and mmtk-core works. You also need to let mmtk-core CI know that the mmtk-core PR should use this JikesRVM binding branch. Like the comments here: https://github.com/mmtk/mmtk-core/pull/136#issuecomment-718384696

Please let me know if any of these confuses you. I will explain in more details.

Comment (qinsoon) [reply mmtk/Cargo.toml:18]: @javadamiri I can do a separate PR to remove the submodule. If you think that is probably better, you can just revert changes around submodules, and I will fix them in a following PR.

Comment (javadamiri) [reply mmtk/Cargo.toml:18]: > You would need a `rev` to specify the revision. You can check the `Cargo.toml` for OpenJDK: https://github.com/mmtk/mmtk-openjdk/blob/7dd5ef78f537b197700feec913890e7d415b16dc/mmtk/Cargo.toml#L24
> 
> With this change, the CI config needs to be updated as well, as the CI runner cannot pull the repo by `ssh`. Check the step here on OpenJDK CI (https://github.com/mmtk/mmtk-openjdk/blob/7dd5ef78f537b197700feec913890e7d415b16dc/.github/workflows/ci.yml#L19).
> 
> Also in mmtk-core, there are a few places you would need some change. Check the difference between JikesRVM and OpenJDK in mmtk-core CI, and make the JikesRVM steps similar to the OpenJDK's.
> 
> Make sure that CI on both mmtk-jikesrvm and mmtk-core works. You also need to let mmtk-core CI know that the mmtk-core PR should use this JikesRVM binding branch. Like the comments here: [mmtk/mmtk-core#136 (comment)](https://github.com/mmtk/mmtk-core/pull/136#issuecomment-718384696)
> 
> Please let me know if any of these confuses you. I will explain in more details.

I can do it

**Pull request merged**